src/DependencyInjection/Configuration.php

```diff
@@ -633,8 +633,8 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->end()
                         ->arrayNode('javascripts')
                             ->defaultValue([
-                                'bundles/sonataadmin/app.js',
-                                'bundles/sonataform/app.js',
+                                'bundles/sonataadmin/app.js?v=4.36',
+                                'bundles/sonataform/app.js?v=2.0',
```

Comment:
The suffix on the sonataform feel wrong since SonataAdmin doesn't hard-lock the sonataform version which could be from 2.0 to 2.4.1 currently.

So
- the suffix here and the version from sonata form won't be synchronized
- If there is a JS/Css change in sonata form, it doesn't imply a version bump on sonataAdmin side